beacon-chain/types/block.go

```diff
@@ -192,14 +196,27 @@ func (b *Block) isAttestationValid(attestationIndex int, aState *ActiveState, cS
 		return false
 	}
 
-	if attestation.JustifiedSlot > cState.LastJustifiedSlot() {
-		log.Debugf("attestation's last justified slot has to match crystallied state's last justified slot. Found: %d. Want: %d",
+	if !(attestation.JustifiedSlot <= cState.LastJustifiedSlot()) {
+		log.Debugf("attestation's justified slot has to be earlier or equal to crystallied state's last justified slot. Found: %d. Want <=: %d",
 			attestation.JustifiedSlot,
 			cState.LastJustifiedSlot())
 		return false
 	}
 
-	// TODO(#468): Validate last justified block hash matches in the crystallizedState.
+	hash := [32]byte{}
+	copy(attestation.JustifiedBlockHash[:], hash[:32])
+	blockInChain, err := chain.ContainsBlock(hash)
+	if err != nil {
+		//need to figure out what would be the "correct" way of going about a database error, how should we terminate?
+		log.Debugf("unable to determine if attestation justified block is in the DB: %s", err)
+		return false
+	}
+
+	if !blockInChain {
+		log.Debugf("attestation's justified block hash has to be in the current chain.  Justified block hash: %v",
```

Comment:
Can you add more to this log? 

ex.
`The attestion's justifed block hash has to be in the current chain, but was not found`